tests/clipboard/clipboard_plugin.test.ts

```diff
@@ -1869,6 +1871,128 @@ describe("clipboard: pasting outside of sheet", () => {
     expect(model.getters.getSelectedZones()).toEqual([toZone("B2:AA2")]);
   });
 
+  test("Ctrl+D on cell(s) of edge row should do nothing", async () => {
```

Comment:
don't forget to adapt the test titles ;-) the keyboard shortcuts don't belong here

Reply:
Changed the titles :+1: